docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -322,6 +322,80 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[dateAddDateQuarters]
 include-tagged::{sql-specs}/docs/docs.csv-spec[dateAddDateMinutes]
 --------------------------------------------------
 
+[[sql-functions-datetime-diff]]
+==== `DATE_DIFF/DATEDIFF/TIMESTAMPDIFF/TIMESTAMP_DIFF`
```

Comment:
Either the names with underscore come first or not (last based on the alphabetical sorting) or not but it should be consistent.

Reply:
yep, thx! fixed also for TIMESTAMPADD/TIMESTAMP_ADD.